content/guides/references/configuration.md

```diff
@@ -145,177 +177,161 @@ file.
 
 The Node version is used in Cypress to:
 
-- Build files in the
-  [integrationFolder](/guides/references/configuration#Folders-Files).
 - Build files in the
   [supportFile](/guides/references/configuration#Folders-Files).
 - Execute code in the
   [pluginsFile](/guides/references/configuration#Folders-Files).
 
-<DocsImage src="/img/guides/test-runner-settings-nodejs-version.jpg" alt="Node version in Settings in Test Runner" ></DocsImage>
+<DocsImage src="/img/guides/test-runner-settings-nodejs-version.jpg" alt="Node version in Settings in Cypress App" ></DocsImage>
 
 ### Experiments
 
 Configuration might include experimental options currently being tested. See
 [Experiments](/guides/references/experiments) page.
 
-## Overriding Options
-
-Cypress gives you the option to dynamically alter configuration values. This is
-helpful when running Cypress in multiple environments and on multiple developer
-machines.
-
-This gives you the option to do things like override the `baseUrl` or
-environment variables.
+## Testing Type-Specific Options
 
-### Command Line
+You can provide configuration options for either E2E or Component Testing by
+creating `e2e` and `component` objects inside your Cypress configuration.
 
-When [running Cypress from the Command Line](/guides/guides/command-line) you
-can pass a `--config` flag.
+### e2e
 
-#### Examples:
+These options are available to be specified inside the `e2e` configuration
+object:
 
-```shell
-cypress open --config pageLoadTimeout=30000,baseUrl=https://myapp.com
-```
+| Option               | Default                               | Description                                                                                                                                                                                         |
+| -------------------- | ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `setupNodeEvents`    | `null`                                | Function in which node events can be registered and config can be modified. Takes the place of the (deprecated) plugins file. [Please read the notes for examples on using this.](#setupNodeEvents) |
+| `supportFile`        | `cypress/support/e2e.{js,jsx,ts,tsx}` | Path to file to load before test files load. This file is compiled and bundled. (Pass `false` to disable)                                                                                           |
+| `specPattern`        | `cypress/e2e/**/*.cy.{js,jsx,ts,tsx}` | A String or Array of glob patterns of the test files to load.                                                                                                                                       |
```

Comment:
It was noted that we are still maintaining the terminology `spec` for a file with tests
```suggestion
| `specPattern`        | `cypress/e2e/**/*.cy.{js,jsx,ts,tsx}` | A String or Array of glob patterns of the spec files to load.                                                                                                                                       |
```